pyanaconda/modules/network/nm_client.py

```diff
@@ -352,6 +367,18 @@ def _update_wired_connection_with_s390_settings(connection, s390cfg):
         s_wired.props.s390_options = opts_dict
 
 
+def _create_new_connection(network_data, device_name):
+    con_uuid = NM.utils_uuid_generate()
+    con = NM.SimpleConnection.new()
+    s_con = NM.SettingConnection.new()
+    s_con.props.uuid = con_uuid
+    s_con.props.id = device_name
+    s_con.props.interface_name = device_name
```

Comment:
I don't know much about netowrking, but won't naming a connection same as the device be confusing?

Reply:
This is a convention we've been sticking to for quite some time in the installer.